.github/workflows/ci_pkg-install.yml

```diff
@@ -102,3 +102,49 @@ jobs:
         with:
           pkg-name: "lightning"
           pip-flags: "-U --pre --find-links ../pypi/"
+
+  install-meta-pypi:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      # max-parallel: 1
+      matrix:
+        os: [ubuntu-20.04, macOS-10.15, windows-2019]
+        python-version: [3.8]  # , 3.9
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: Dowload package
+        run: |
+          pip install -q fire requests
+          for pkg in 'app' 'pytorch' ; do
+            python .actions/assistant.py download-package "$pkg" --folder pypi
+          done
+          ls -lh pypi/
+
+      - name: Unzip packages
+        working-directory: pypi
+        run: for file in `ls *.gz`; do tar -xzf $file; done
+      - name: Show upacked pkgs
+        run: |
+          sudo apt install -y tree
+          tree pypi/ -L 3
+
+      - name: Miror source
+        run: |
+          pip install -q fire requests
+          python .actions/assistant.py mirror-pkg2source pypi src
+          tree src/ -L 3
```

Comment:
I'm not quite sure what the assistant script tries to resolve here (maybe because I don't know much about how the wrapping package `src/lightning/` will be distributed).

What's the expected result from `tree src/ -L 3`?

Reply:
Tree is rather debug view inside...

Otherwise the point of copy is that meta package need to be created from adequate sources so in case we release PL only we refer App as stable which is not necessarily in code base at this time but the one on pypi...